### ISSUE TITLE
Search parcels by line or polygon : interruption by the identify tool

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/mapstore2-cadastrapp#95]
![select_with_line](https://user-images.githubusercontent.com/6370443/109141955-94ae1580-775e-11eb-81d3-8d7c2f0b74c2.gif)

when I want to draw a second selection with the line or polygon tool (in the same tab or not) I cannot : it is the identify tool wiwh comes.

**Comment by tdipisa** [comment]
@MaelREBOUX it is a problem I replicated only in your installation [here](https://portail-test.sig.rennesmetropole.fr/mapstore/#/context/Cadastre-urbanisme): in geosolutions DEV instance it is working properly. I will ask developers to check for a possible cause.

**Comment by MaelREBOUX** [comment]
sad news.

@landryb or @jusabatier do you replicate ?

**Comment by landryb** [comment]
cant reproduce with https://ids.craig.fr/mapstore/#/context/cadastre - @tdipisa maybe recheck with/without the identify tool enabled (& activated by default?) in the context ?

**Comment by offtherailz** [comment]
Hi, I tried to check and debug  [here](https://portail-test.sig.rennesmetropole.fr/mapstore/#/context/Cadastre-urbanisme) your instance but it is very hard. I have a lot of slow downs and network erros (maybe CI)
For what I've seen there is an issue using drawSupport, and in fact also the annotations do not seem to work well. 
I think there is another extension is actively listening and stopping the draw, and this breaks interaction with the map by the other plugins.

Can I suggest to install the latest version first (I noticed the installed version contains some bugs that has been already fixed) and try to remove the other extensions from the environment, to check if they are causing the problem.
Then include them incrementally to find out the responsible. I see there is also a verbose logging by some of them that could be avoided (it slows down a little the application, too).
